indexing-service/src/test/java/io/druid/indexing/overlord/TestRemoteTaskRunnerConfig.java

```diff
@@ -33,7 +33,8 @@ public Period getTaskAssignmentTimeout()
   @Override
   public long getMaxZnodeBytes()
   {
-    return 1000;
+    // make sure this is large enough, otherwise RemoteTaskRunnerTest might fail unexpectedly
```

Comment:
Basically, the addition of `IndexSpec` to the tuning config for the `RealtimeTask` pushed the zNode data size beyond 1000 bytes and caused the test to fail. The test config was set to 1000 bytes, even though the parent config requires a minimum of 10kB